test/Azure.Functions.Cli.Tests/ActionsTests/StartHostActionTests.cs

```diff
@@ -72,6 +72,29 @@ public async Task CheckNonOptionalSettingsDoesntThrowOnMissingAzureWebJobsStorag
             exception.Should().BeNull();
         }
 
+        [Fact]
+        public async Task CheckNonOptionalSettingsDoesntThrowOnMissingAzureWebJobsStorageWhenFlagIsSet()
+        {
+            var fileSystem = GetFakeFileSystem(new[]
+   {
+                ("x:\\folder1", "{'bindings': [{'type': 'httpTrigger'}]}"),
+                ("x:\\folder2", "{'bindings': [{'type': 'httpTrigger'}]}")
```

Comment:
I think you might wanna change one of these to `timerTrigger` or some other trigger that requires storage check?

Reply:
Good call